independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/dockerfiles/base/Dockerfile-layout.include.qute

```diff
@@ -18,31 +18,18 @@
 #
 # Then run the container using :
 #
-# docker run -i --rm -p 8080:8080 -p 5005:5005 -e JAVA_ENABLE_DEBUG="true" quarkus/{project.artifact-id}-{type}
+# docker run -i --rm -p 8080:8080 quarkus/{project.artifact-id}-{type}
 #
 ###
-FROM {#insert image /}
-{#insert args /}
+FROM registry.access.redhat.com/ubi8/openjdk-{java.version}-runtime:1.10
```

Comment:
I thought I put this comment but can't find it. Does this not break former cli running these as they expect image to be passed in ?

Reply:
@maxandersen, I'm not contextualized with this cli.
which cli are you talking about? and which command would impact?

Reply:
We are safe, each codestart is kept as a unit.

Reply:
So if using quarkus 2.6 cli or mvn plugin how does it know how to pass in "image" to the former version ?

Is this defined somewhere else ?

Reply:
The image is part of the template so no need for data. The `{java.version}` already exists in previous versions